prijateli_tree/app/routers/games.py

```diff
@@ -198,6 +206,22 @@ def get_previous_answers(
     }
 
 
+def language_check(player_lang, languages):
+    """
+    Helper function to ensure that the language is in the languages dict
+    """
+    if player_lang not in languages:
+        player_lang = player_lang.lower()
+
+    if player_lang not in languages:
```

Comment:
Every player has a language attached to them, there shouldn't be a case where this is applicable.
<img width="619" alt="Screenshot 2023-11-24 at 1 28 42 PM" src="https://github.com/prijatelilab/PrijateliTree/assets/5885605/e3425864-7b60-4f25-b033-b04e253af4e5">

You shouldn't need this function.